project/AppDependencies.scala

```diff
@@ -14,7 +14,7 @@ object AppDependencies {
     "com.typesafe.play"       %% "play-json-joda"                    % "2.9.2",
     "org.julienrf"            %% "play-json-derived-codecs"          % "7.0.0",
     "uk.gov.hmrc"             %% "json-encryption"                   % "5.1.0-play-28",
-    "uk.gov.hmrc"             %% "time"                              % "3.25.0",
+    "uk.gov.hmrc"             %  "time_2.12"                         % "3.25.0",
```

Comment:
This looks like Scala 2.12 - what was the reason for this change?

Reply:
Yes, there is the latest available dependency version available, we are updating to 2.13, therefore it need to declared explicitly.